cypress/support/commands.js

```diff
@@ -12,59 +30,162 @@ import { AUTO_SAVE_PAUSE, LOAD_PAGE_PAUSE } from '../constants/constants';
 // https://on.cypress.io/custom-commands
 // ***********************************************
 //
-Cypress.Commands.add('offlineVisit', (mode = DEFAULT_MODE) => {
-  cy.visit('/', {
-    qs: {
-      offline: true,
-      spaceId: '5b56e70ab253020033364411',
-      appInstanceId: '6156e70ab253020033364411',
-      mode,
-      userId: '5b56e70ab253020033364416',
-    },
-  });
-  cy.wait(LOAD_PAGE_PAUSE);
+Cypress.Commands.add(
+  'onlineVisit',
+  ({
+    mode = DEFAULT_MODE,
+    spaceId = '5b56e70ab253020033364411',
+    appInstanceId = '6156e70ab253020033364411',
+    userId = '5b56e70ab253020033364416',
+  } = {}) => {
+    cy.visit('/', {
+      qs: {
+        spaceId,
+        appInstanceId,
+        mode,
+        userId,
+        dev: true,
+      },
+    });
+    cy.wait(LOAD_PAGE_PAUSE);
+  }
+);
+
+Cypress.Commands.add('enterStudentResponse', ({ text, save = false }) => {
+  const input = cy.get(STUDENT_INPUT);
+  if (!text || text === '') {
+    input.clear().should('be.empty');
+  } else {
+    input.type(text).should('have.value', text);
+  }
+
+  if (save) {
+    cy.saveAnswer();
+  }
 });
-Cypress.Commands.add('onlineVisit', (mode = DEFAULT_MODE) => {
-  cy.visit('/', {
-    qs: {
-      spaceId: '5b56e70ab253020033364411',
-      appInstanceId: '6156e70ab253020033364411',
-      mode,
-      userId: '5b56e70ab253020033364416',
-      dev: true,
-    },
-  });
-  cy.wait(LOAD_PAGE_PAUSE);
+
+Cypress.Commands.add('clearStudentResponse', () => {
+  cy.get(STUDENT_INPUT).clear();
 });
 
+const deleteAnswers = () => {
```

Comment:
Why is this not added as a Cypress command?

Reply:
It is part of `clearData` which is a cypress command. I don't think this function will be reusable since it deletes any answer (the first found within the page).